reports/distributions/contrib.yaml

```diff
@@ -118,6 +118,7 @@ components:
         - transform
     provider:
         - aes
+        - googlesecretmanagerprovider
```

Comment:
This has made me realize that in the initial PR I didn't notice `provider` is in the `type` field for `metadata.yaml` of this component. I don't think it should, as other components of any type don't include the type of component in the type name. So we should change that to `googlesecretmanager` along with any other necessary changes from that

Reply:
Done